search/query_language_whitebox_test.go

```diff
@@ -0,0 +1,236 @@
+package search
+
+import (
+	"encoding/json"
+	"runtime/debug"
+	"testing"
+
+	c "github.com/fabric8-services/fabric8-wit/criteria"
```

Comment:
Minor: why do you add these alias ? I am not against using alias but I think the current default alias are better while reading the code. You can spot the destination pkg, we don't use it in other places unless there are conflicts